modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -236,25 +236,30 @@
 	name = "12g Buckshot Drum"
 	desc = "An additional 8-round buckshot magazine for use with the Bulldog shotgun. Front towards enemy."
 	item = /obj/item/ammo_box/magazine/m12g
+	purchasable_from = ALL
 
 /datum/uplink_item/ammo/shotgun/dragon_traitor
 	name = "12g Dragon's Breath Drum"
 	desc = "An alternative 8-round dragon's breath magazine for use in the Bulldog shotgun. \
 			'I'm a fire starter, twisted fire starter!'"
 	item = /obj/item/ammo_box/magazine/m12g/dragon
+	purchasable_from = ALL
```

Comment:
Why do these two have no price on them?

Reply:
the parent, `/datum/uplink_item/ammo/shotgun`, has the price set to 2